util/file.py

```diff
@@ -473,3 +494,33 @@ def line_count(infname):
 def touch(fname, times=None):
     with open(fname, 'a'):
         os.utime(fname, times)
+
+def touch_empty(fname):
+    '''Make each file in fnames a zero-length file with the current timestamp.'''
+    with open(fname, 'w'):
+        pass
+
+def dump_file( fname, value ):
+    """store string in file"""
+    with open( fname, 'w' )  as out: 
+        out.write( str( value ) )
+
+def slurp_file( fname ):
```

Comment:
I'm a little worried that `slurp_file` might get used a lot when it shouldn't be.. I realize it's convenient at times to take a quick pull of inputs that we know to be O(1) in size... but putting it here almost makes it to convenient for developers to avoid good iterator practice.